### DISCUSSION TITLE
How to avoid bibliography getting a number

### DISCUSSION THREAD

**Discussion by patrick-kidger** [typst/typst#1055] [General]
Right now I'm using the following to add some padding below each of my headings:
```typst
show heading: x => pad(bottom: 5pt)[#counter(heading).display() #x.body]
```
this has the unfortunate side-effect that my bibliography also has a number. (And moreover, the number is the same as the last section, i.e. it hasn't incremented).

How might this be avoided?

- I tried using an if statement `if x.body == "Bibliography"`, but this doesn't seem to work as `x.body` is a content type, not a string type, and so far as I know there's no way to turn content back into strings.
- I tried using an if statement `if x.body ==  ["Bibliography"]`, but this likewise failed. I'm guessing content types might be compared equal based on identity rather than structure.

Thanks!

**Comment by PgBiel** [comment]
Try `panic(x.body)` to see what it consists of (add the `show` right before bibliography to see). I think `x.body.text == "Bibliography"` could work here.

**Comment by laurmaedje** [comment]
You can make the counter display conditional on the heading being numbered like this: `if x.numbering != none { counter(heading).display() }`. That should do it.

**Reply by patrick-kidger** [reply]
Aha, that works! Thank you.
